pkg/virt-controller/watch/migration.go

```diff
@@ -406,6 +406,10 @@ func (c *MigrationController) updateStatus(migration *virtv1.VirtualMachineInsta
 
 	// Remove the finalizer and conditions if the migration has already completed
 	if migration.IsFinal() {
+		// store the finalized migration state data from the VMI status in the migration object
+		migrationCopy.Status.MigrationState = vmi.Status.MigrationState
```

Comment:
No need for a deepcopy?

Reply:
@jean-edouard I think that `migrationCopy` is itself the copied object